appservice/src/main/java/nu/yona/server/subscriptions/rest/UserController.java

```diff
@@ -712,9 +660,8 @@ private void addUserPhotoLinkIfPhotoPresent(UserResource userResource)
 
 		private void addAppleMobileConfigLink(UserResource userResource)
```

Comment:
Is there a task to remove this as well?

Reply:
No. It isn't strictly necessary to take it out here, as it's based on the requesting device. So, we could stay backward compatible and keep it here. Alternatively, we could add the link to the device and ask the app to take it from there.

Reply:
See https://jira.yona.nu/browse/YD-544